File: app/src/main/java/com/real0168/baseproject/MainActivity.java
package com.real0168.baseproject;

import android.os.Bundle;
import android.support.annotation.Nullable;

import com.real0168.base.BaseActivity;

public class MainActivity extends BaseActivity {
    @Override
    public int getLayoutID() {
        return R.layout.activity_main;
    }

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }
}

File: app/src/main/java/com/real0168/baseproject/MainActivity.kt
package com.real0168.baseproject

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import com.real0168.base.BaseActivity

class MainActivity : BaseActivity() {
    override fun getLayoutID(): Int {
        return R.layout.activity_main
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }
}
